tests/serac_dynamic_solver.cpp

```diff
@@ -193,7 +193,7 @@ TEST(dynamic_solver, dyn_linesearch_solve)
   MPI_Barrier(MPI_COMM_WORLD);
 
   // Open the mesh
-  std::string mesh_file = std::string(SERAC_REPO_DIR) + "/data/beam-hex.mesh";
+  std::string mesh_file = std::string(SERAC_REPO_DIR) + "/data/meshes/beam-hex.mesh";
```

Comment:
NOOOO i missed one?!?! :shame:

Reply:
This points at the fact we should be running our tests with optional dependencies.

Reply:
I think this one was on me - I think the sundials stuff went in after the inlet stuff.

Reply:
meh.. you can blame me, very broken today. so tired. need breakfast burrito....

Reply:
🌯